api_admin_panel/app/components/output.py

```diff
@@ -0,0 +1,87 @@
+import reflex as rx
+
+from app.states.queries import QueryAPI
+from app.components.drawer import render_drawer
+
+
+def create_table_header(title: str):
+    return rx.table.column_header_cell(title)
+
+
+def create_query_rows(data: dict[str, str]):
+    def fill_rows_with_data(data_):
+        return rx.table.cell(
+            f"{data_[1]}",
+            on_double_click=QueryAPI.display_selected_row(data),
```

Comment:
Maybe we can change the double click for a single one, hard to notice you can do that unless there's some hint anywhere else